kocho.yml.example

```diff
@@ -28,6 +28,8 @@ certificate: <insert ssl certificate arn here>
 # yochu-k8s-version: <k8s version>
 # yochu-rkt-version: <rkt version>
 
+# Use ignition
```

Comment:
do we want to add a link to Ignition here? Just to give readers a hint what this is about?